### ISSUE TITLE
Velocity doesn't add new options to the velocity.toml config file

### ISSUE THREAD

**Issue by seb303** [PaperMC/Velocity#733]
After updating to velocity-3.1.2-SNAPSHOT-147 I needed to change option `force-key-authentication` but saw that it was missing.  After renaming the config file and restarting Velocity to generate a new one, I notice there were quite a few options missing from the old config file.

I would expect Velocity to add in any new options and not have to manually regenerate the file then copy across existing settings.

Our dev server is running on Windows 10, in case that makes a difference.

**Comment by zefir-git** [comment]
Doing the same I got the following error
```
[07:29:16 ERROR]: Unable to read/load/save your velocity.toml. The server will shut down.
java.io.IOException: Is a directory
        at sun.nio.ch.FileDispatcherImpl.read0(Native Method) ~[?:?]
        at sun.nio.ch.FileDispatcherImpl.read(FileDispatcherImpl.java:48) ~[?:?]
        at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:330) ~[?:?]
        at sun.nio.ch.IOUtil.read(IOUtil.java:296) ~[?:?]
        at sun.nio.ch.IOUtil.read(IOUtil.java:273) ~[?:?]
        at sun.nio.ch.FileChannelImpl.read(FileChannelImpl.java:232) ~[?:?]
        at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:65) ~[?:?]
        at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:107) ~[?:?]
        at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:101) ~[?:?]
        at sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:270) ~[?:?]
        at sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:313) ~[?:?]
        at sun.nio.cs.StreamDecoder.read(StreamDecoder.java:188) ~[?:?]
        at java.io.InputStreamReader.read(InputStreamReader.java:177) ~[?:?]
        at java.io.BufferedReader.fill(BufferedReader.java:162) ~[?:?]
        at java.io.BufferedReader.readLine(BufferedReader.java:329) ~[?:?]
        at java.io.BufferedReader.readLine(BufferedReader.java:396) ~[?:?]
        at java.nio.file.Files.readAllLines(Files.java:3415) ~[?:?]
        at java.nio.file.Files.readAllLines(Files.java:3453) ~[?:?]
        at com.velocitypowered.proxy.config.VelocityConfiguration.read(VelocityConfiguration.java:482) ~[velocity.jar:3.1.2-SNAPSHOT (git-74edac96-b153)]
        at com.velocitypowered.proxy.VelocityServer.doStartupConfigLoad(VelocityServer.java:308) ~[velocity.jar:3.1.2-SNAPSHOT (git-74edac96-b153)]
        at com.velocitypowered.proxy.VelocityServer.start(VelocityServer.java:214) ~[velocity.jar:3.1.2-SNAPSHOT (git-74edac96-b153)]
        at com.velocitypowered.proxy.Velocity.main(Velocity.java:64) ~[velocity.jar:3.1.2-SNAPSHOT (git-74edac96-b153)]
```
linux

**Comment by 4drian3d** [comment]
> Doing the same I got the following error
> 
> ```
> [07:29:16 ERROR]: Unable to read/load/save your velocity.toml. The server will shut down.
> java.io.IOException: Is a directory
>         at sun.nio.ch.FileDispatcherImpl.read0(Native Method) ~[?:?]
>         at sun.nio.ch.FileDispatcherImpl.read(FileDispatcherImpl.java:48) ~[?:?]
>         at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:330) ~[?:?]
>         at sun.nio.ch.IOUtil.read(IOUtil.java:296) ~[?:?]
>         at sun.nio.ch.IOUtil.read(IOUtil.java:273) ~[?:?]
>         at sun.nio.ch.FileChannelImpl.read(FileChannelImpl.java:232) ~[?:?]
>         at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:65) ~[?:?]
>         at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:107) ~[?:?]
>         at sun.nio.ch.ChannelInputStream.read(ChannelInputStream.java:101) ~[?:?]
>         at sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:270) ~[?:?]
>         at sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:313) ~[?:?]
>         at sun.nio.cs.StreamDecoder.read(StreamDecoder.java:188) ~[?:?]
>         at java.io.InputStreamReader.read(InputStreamReader.java:177) ~[?:?]
>         at java.io.BufferedReader.fill(BufferedReader.java:162) ~[?:?]
>         at java.io.BufferedReader.readLine(BufferedReader.java:329) ~[?:?]
>         at java.io.BufferedReader.readLine(BufferedReader.java:396) ~[?:?]
>         at java.nio.file.Files.readAllLines(Files.java:3415) ~[?:?]
>         at java.nio.file.Files.readAllLines(Files.java:3453) ~[?:?]
>         at com.velocitypowered.proxy.config.VelocityConfiguration.read(VelocityConfiguration.java:482) ~[velocity.jar:3.1.2-SNAPSHOT (git-74edac96-b153)]
>         at com.velocitypowered.proxy.VelocityServer.doStartupConfigLoad(VelocityServer.java:308) ~[velocity.jar:3.1.2-SNAPSHOT (git-74edac96-b153)]
>         at com.velocitypowered.proxy.VelocityServer.start(VelocityServer.java:214) ~[velocity.jar:3.1.2-SNAPSHOT (git-74edac96-b153)]
>         at com.velocitypowered.proxy.Velocity.main(Velocity.java:64) ~[velocity.jar:3.1.2-SNAPSHOT (git-74edac96-b153)]
> ```
> 
> linux

The version of Velocity you used had a bug, try the [latest version](https://papermc.io/downloads#Velocity), the IOException has been fixed

**Comment by Xernium** [comment]
Fixed by #772
